File: java/src/main/leetcode/TwoSum.java
package main.leetcode;


public class TwoSum {

    public  static void main(String [] args){

        int nums []= {2, 7, 11, 15};
        int target = 9;
       int[] arr=twoSum( nums,  target);
      System.out.println(arr);
    }
    public static  int[] twoSum(int[] nums, int target) {
        for(int i=0;i<nums.length;i++){
            for(int j=0;j<nums.length;j++){
                if(i!=j&&nums[i]+nums[j]==target){
                    int arr[]=new int[2];
                    arr[0]=i;
                    arr[1]=j;
                    return arr ;

                }
            }
        }
        return null;
    };
}

File: leetcode/twoSum.js
//167 两数之和 II - 输入有序数组
var twoSum = function(numbers, target) {
    let arr=numbers.filter(item=>item<=target-numbers[0])
    for(let i=0;i<arr.length;i++){
        let leftArr=arr.slice(i+1)
        if(leftArr.indexOf(target-arr[i])>-1){
            return [i+1,i+leftArr.indexOf(target-arr[i])+2]
        }
    }
    return []
};
console.log(twoSum([-1,0],-1))